### PR TITLE
Added New Team Members to Abouts page and updated description

### PR DESCRIPTION
**Pull request by oscarso2000**
Minor Updates to About.tsx and About.css

1. Revised Description + Added collaboration mention with Cornell Data Science
2. Centered text and team member information.
3. Revised Authors to Co-Founders
4. Added 6 new team-members (2 images have not been uploaded yet)
5. Moved "Source code for GitHub" to the bottom along with v1 and v2 links for better aesthetics. 

Further Note: v2.mycourseindex.com will now be the old MCI code up till Sep 17, 2020. 

**Do not accept this request, it is incomplete**

### CODE REVIEW

**Comment by oscarso2000** [comment]
Fixed One Addition Line in Abouts.css "margin-top: 35%" as there was too much white space between top of page and "about" description.

**Comment by oscarso2000** [comment]
All Github links work and all pictures show up.

**Comment by oscarso2000** [comment]
Screen Shot 2020-09-18 at 2 55 48 PM

**Comment by oscarso2000** [comment]
**Reviews Requested**

File: client/src/style/About.css [hunk 1]
```diff
@@ -15,25 +15,35 @@
   color: #ffffff;
   padding-left: 5.5px;
 }
-.center {
+.centerAbout {
```
Comment (bayoumi17m) [inline client/src/style/About.css:18]: Why the change in the naming scheme? This adds complexity since it's just a center and it must be in the about section since it is named About.css.

This is more of a stylistic choice but I am just trying to understand your objective here

File: client/src/components/About.tsx [hunk 2]
```diff
@@ -13,41 +21,95 @@ export const About: React.StatelessComponent = (): JSX.Element => (
...
-        <div className="center">
+        <div className="centerAbout">
             <h4>About</h4>
-            <h4><a href="https://github.com/oscarso2000/mycourseindex">Source code on Github</a></h4>
-            <p> The authors of the MyCourseIndex project are: </p>
 
-            <div className="item">
-                <img src={Magd} />
-                <span className="caption"><a href="https://github.com/bayoumi17m">Magd Bayoumi (mb2363)</a></span>
+            <p>MyCourseIndex was initially a project started for CS/INFO 4300: Language and Information that acts as an essential search engine for Cornell students and their courses. This search gathers all information from Piazza posts to Textbook and Notes Resources and returns valid results for the student to utilize. It is now being maintained and improved by members of <a href="https://cornelldata.science">Cornell Data Science Project Team</a>.</p>
```
Comment (bayoumi17m) [inline client/src/components/About.tsx:27]: This line is much too long. It needs to be broken into broken down so that it's readable within an editor (without wrapping).

We also should update the description with more about the new product and less about the class etc. We want to focus on the product and team as much as possible

File: client/src/style/CourseSelection.css [hunk 3]
```diff
@@ -11,6 +11,17 @@
...
+.center {
+  position: absolute;
+  margin-top: 22%;
+  left: 50%;
+  margin-right: -50%;
+  transform: translate(-50%, -50%);
+  width: 45vw;
+  color: #ffffff;
+}
+
```
Comment (bayoumi17m) [inline client/src/style/CourseSelection.css:24]: What impact does this have on the mapped cards from:

https://github.com/oscarso2000/MyCourseIndex/blob/76689cab401bb649d37a69099a6921ae315683c2/client/src/components/CourseSelection.tsx#L31-L48

**Review by bayoumi17m — COMMENTED** [review]
Overall looks good, just looking for some clarification 😁

File: client/src/style/About.css (hunk 1 above)
Comment (oscarso2000) [reply client/src/style/About.css:18]: The About.css was somehow impacting and creating a bug in courseselection so I had to change it around.

File: client/src/components/About.tsx (hunk 2 above)
Comment (oscarso2000) [reply client/src/components/About.tsx:27]: Should i just hit enter at the end of every 130 characters or something since its wrapped in the p-text body already? 

Yea I just want the pictures up and running, we can change the description anytime.

File: client/src/style/CourseSelection.css (hunk 3 above)
Comment (oscarso2000) [reply client/src/style/CourseSelection.css:24]: the margin-top is the most important cuz there was a really weird bug that was shifting the card modals around so i had to find the exact location given screen-resizing to put the course cards.

File: client/src/components/About.tsx (hunk 2 above)
Comment (bayoumi17m) [reply client/src/components/About.tsx:27]: Ok, if that's the case, we can revisit this later

File: client/src/style/About.css (hunk 1 above)
Comment (bayoumi17m) [reply client/src/style/About.css:18]: And when CourseSelection has a center attribute, does this issue still occur?

File: client/src/style/CourseSelection.css (hunk 3 above)
Comment (bayoumi17m) [reply client/src/style/CourseSelection.css:24]: Screenshot to show the impact and what the original thing was?

File: client/src/style/About.css (hunk 1 above)
Comment (oscarso2000) [reply client/src/style/About.css:18]: no, but might as well make the diiv names better to understand

Comment (bayoumi17m) [reply client/src/style/About.css:18]: But isn't center clear? It's more complex by adding the about since that implies there is another center here.

File: client/src/style/CourseSelection.css (hunk 3 above)
Comment (oscarso2000) [reply client/src/style/CourseSelection.css:24]: Screen Shot 2020-09-18 at 3 29 00 PM

Comment (oscarso2000) [reply client/src/style/CourseSelection.css:24]: Screen Shot 2020-09-18 at 3 30 47 PM
fixed version w/ css

File: client/src/style/About.css (hunk 1 above)
Comment (oscarso2000) [reply client/src/style/About.css:18]: ahh ok ill change it back to center then.

Comment (oscarso2000) [reply client/src/style/About.css]: @bayoumi17m Changed. Please review again.

**Comment by oscarso2000** [comment]
**Review requested again**

**Review by bayoumi17m — DISMISSED** [review]
Looks good to me! 😁

**Review by yangkt — APPROVED** [review]

**Review by shljessie — APPROVED** [review]

File: client/src/components/About.tsx [hunk 4]
```diff
@@ -15,39 +23,93 @@ export const About: React.StatelessComponent = (): JSX.Element => (
...
+
+            <div className="centertext">
+                <p> <b>Co-Founders:</b> </p>
             </div>
-            <div className="item">
-                <img src={Jenna} />
-                <span className="caption"><a href="https://github.com/jek343">Jenna Kressin (jek343)</a></span>
+            <div className="innerimage">
+                <div className="item">
+                    <img src={Magd} />
```
Comment (shljessie) [inline client/src/components/About.tsx:34]: It's not something serious but I'm getting this javascript warning in my terminal. I think that we should add an alt text here just to follow the convention !
Screen Shot 2020-09-19 at 1 05 35 PM

File: client/src/components/About.tsx [hunk 5]
```diff
@@ -15,39 +23,93 @@ export const About: React.StatelessComponent = (): JSX.Element => (
...
+                <div className="item">
+                    <img src={John} />
+                    <span className="caption"><a href="https://github.com/jodonnell77">John O'Donnell (jro79)</a></span>
+                </div>
             </div>
+
             <p></p>
-            <p>MyCourseIndex is a project for CS/INFO 4300: Language and Information that acts as an essential search engine for Cornell students and their courses. This search gathers all information from Piazza posts to Textbook and Notes Resources and returns valid results for the student to utilize. </p>  
+
             <a href="https://v1.mycourseindex.com" target="_blank">
```
Comment (shljessie) [inline client/src/components/About.tsx:94]: Also for this section! 
rel="noopener noreferrer" alongside target should be added!

Screen Shot 2020-09-19 at 1 10 59 PM

Comment (bayoumi17m) [reply client/src/components/About.tsx:94]: @shljessie Good catch! I missed this, @oscarso2000 this is an actual issue. Depending on the site it links to, it may not be as severe but should be fixed.

Comment (bayoumi17m) [reply client/src/components/About.tsx:94]: Below is the only reference I found, but Jessie's has a reference at line 44 which I am unable to find in the current branch. 
https://github.com/oscarso2000/MyCourseIndex/blob/b657c586e1179f5f6683ed40657b4272f0bc146c/client/src/components/About.tsx#L93-L108

**Review by bayoumi17m — CHANGES_REQUESTED** [review]
Correct the `noreferrer noopener` issue in the <a></a> tags within `About.tsx`

**Review by yangkt — APPROVED** [review]

File: client/src/components/About.tsx (hunk 5 above)
Comment (oscarso2000) [reply client/src/components/About.tsx]: Good catch! I've fixed it.
